### PR TITLE
Update KFServing roadmap for Q3/Q4

### PR DESCRIPTION
**Pull request by yuzisun**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,22 +1,46 @@
...
 
-Objective: "Reduce Total Cost of Ownership when deploying multiple underutilized models."
-* GPU Sharing 
-    * Reduce TCO by enabling models of the same framework and version to be co-hosted in a single model server.
+Objective: "Increase throughput for the inference service"
+* Adaptive batching support(Inspur)
+    * Queue and batch requests to increase throughput.
 
-### v0.3 Stability (ETA: Dec 15, 2019)
+* Bias, Skew, and Outlier Detection.
```
Comment (ellistarn) [inline ROADMAP.md:40]: Does this need to be deleted?

File: ROADMAP.md [hunk 2]
```diff
@@ -1,22 +1,46 @@
...
+    * V2 Python Server
+    * Triton inference server V2 prediction API
+    * TorchServe/KFServing integration
+
+Objective: "Reduce Total Cost of Ownership when deploying multiple underutilized models."
+* Container/GPU Sharing
+    * Reduce TCO by enabling models of the same framework and version to be co-hosted in a single model server.
 
-### v0.4 Performance (ETA: Jan 31, 2020)
+### v0.4 Performance (end of Q2)
```
Comment (ellistarn) [inline ROADMAP.md:25]: I'm not sure how much of this performance work was done. I know there was some ad hoc stuff. Should we punt on a Q2 release and just treat 0.4 as a kubeflow cut for kf 1.1?

File: ROADMAP.md [hunk 3]
```diff
@@ -1,22 +1,46 @@
...
 ## 2019 - 2020
+### v0.6 Inference Graph (ETA end of Q4)
+Objective: Enable model serving pipelines with flexible routing graph
+* Inference Router
+    * Model Experimentation.
+    * Ensembling.
+    * Multi Arm Bandit.
+    * Pipeline
+
+### v0.5 KFServing V1Beta1 API and V2 prediction API (ETA: end of Q3)
```
Comment (ellistarn) [inline ROADMAP.md:11]: I like to name them as theses. Consider "API Stabilization and TCO Reduction"

File: ROADMAP.md (hunk 2 above)
Comment (yuzisun) [reply ROADMAP.md:34]: I have put up a PR for the detailed benchmarking https://github.com/kubeflow/kfserving/pull/894, we can call it a 0.4 release

**Pull request merged**